build/KoreBuild.ps1

```diff
@@ -68,6 +68,22 @@ function InstallSharedRuntime([string] $version, [string] $channel)
     }
 }
 
+function BuildTaskProject {
+    $taskProj = "$repoFolder/build/tasks/RepoTasks.csproj"
+    $publishFolder = "$repoFolder/build/tasks/bin/publish/"
+
+    if (!(Test-Path $taskProj)) {
+        return
+    }
+
+    if (Test-Path $publishFolder) {
+        Remove-Item $publishFolder -Recurse -Force
```

Comment:
Does this kill incremental build? Like does leaving the directory there and let `dotnet publish` deal with it work?